conda-forge.yml

```diff
@@ -1,6 +1,6 @@
 provider: {linux_aarch64: azure, linux_ppc64le: azure, win: azure}
 bot:
-  automerge: false
+  inspection: update-grayskull
```

Comment:
I am unsure what this means.

Reply:
It will update the dependencies automatically for us making it easier to review and merge.

See https://twitter.com/condaforge/status/1647933355877445633 for an example.